AgentQnA/kubernetes/helm/rocm-tgi-values.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2025 Advanced Micro Devices, Inc.
+
+# Accelerate inferencing in heaviest components to improve performance
+# by overriding their subchart values
+vllm:
+  enabled: false
+tgi:
+  enabled: true
+  accelDevice: "rocm"
+  image:
+    repository: ghcr.io/huggingface/text-generation-inference
+    tag: "2.4.1-rocm"
+  LLM_MODEL_ID: "meta-llama/Meta-Llama-3-8B-Instruct"
+  MAX_INPUT_LENGTH: "1024"
+  MAX_TOTAL_TOKENS: "2048"
+  USE_FLASH_ATTENTION: "false"
+  FLASH_ATTENTION_RECOMPUTE: "false"
+  HIP_VISIBLE_DEVICES: "0"
+  MAX_BATCH_SIZE: "4"
+  extraCmdArgs: [ "--num-shard","1" ]
+  resources:
+    limits:
+      amd.com/gpu: "1"
+    requests:
+      cpu: 1
+      memory: 16Gi
+  securityContext:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    capabilities:
+      add:
+        - SYS_PTRACE
+  readinessProbe:
+    initialDelaySeconds: 60
+    periodSeconds: 5
+    timeoutSeconds: 1
+    failureThreshold: 120
+  startupProbe:
+    initialDelaySeconds: 60
+    periodSeconds: 5
+    timeoutSeconds: 1
+    failureThreshold: 120
+supervisor:
+  llm_endpoint_url: http://{{ .Release.Name }}-tgi
+  llm_engine: tgi
+  model: "meta-llama/Meta-Llama-3-8B-Instruct"
```

Comment:
Let's align the values with GenAIInfra, https://github.com/opea-project/GenAIInfra/pull/975/files#diff-8c465c490b6932f29f8e7222851ed7ec3cf5b183394d47d4ae16e6747ae3a5acR57